catalog/dags/providers/provider_api_scripts/inaturalist.py

```diff
@@ -328,7 +328,10 @@ def create_preingestion_tasks():
                 python_callable=INaturalistDataIngester.load_catalog_of_life_names,
                 doc_md="Load vernacular taxon names from Catalog of Life",
                 op_kwargs={
-                    "remove_api_files": "{{params.sql_rm_source_data_after_ingesting}}"
+                    "remove_api_files": any(
+                        "{{ params.sql_rm_source_data_after_ingesting }}",
+                        "{{ var.json.SQL_RM_SOURCE_DATA_AFTER_INGESTION}}",
+                    )
```

Comment:
I don't think we can call `any` here because (for complicated Airflow reasons) the `any` gets evaluated when the DAG is parsed, but not when it's run. When the DAG is parsed, the values passed into `any` will be the strings we see here. They're only converted into the actual values when the DAG is actually run. I think this should work:

```suggestion
                    "remove_api_files": "{{ params.sql_rm_source_data_after_ingesting and var.json.SQL_RM_SOURCE_DATA_AFTER_INGESTION }}",
```